src/main/modules/nunjucks/index.ts

```diff
@@ -10,11 +10,16 @@ export class Nunjucks {
 
   enableFor (app: express.Express) {
     app.set('view engine', 'njk')
+    const govUkFrontendPath = path.join(
+      __dirname, '..', '..', '..', '..', 'node_modules',
+      'govuk-frontend'
+    )
     nunjucks.configure([
       path.join(__dirname,'..','..','views'),
+      govUkFrontendPath
     ], {
       autoescape: true,
-      throwOnUndefined: true,
+      //throwOnUndefined: true,
```

Comment:
Follow up PR and remove for now or did you want to fix here?

Reply:
Removed it